pkg/gcp/machine_controller_util.go

```diff
@@ -82,6 +82,10 @@ func (ms *MachinePlugin) CreateMachineUtil(ctx context.Context, machineName stri
 		}
 	)
 
+	if providerSpec.MinCpuPlatform != "" {
```

Comment:
Is there any particular reason for the conditional here instead of using it directly ?

Reply:
Nope.
I've checked and just passing it along should be fine so I'll just do that.